crowbar_framework/app/assets/javascripts/installer.js

```diff
@@ -0,0 +1,86 @@
+$(document).ready(function() {
+  $("body.installer").each(function() {
+    function statusCheck() {
+      $.ajax({
+        type: "GET",
+        dataType: "json",
+        url: Routes.installer_status_path(),
+        success: function(data) {
+          var failed = data.failed;
```

Comment:
I generally dislike to define all these variables. Just access the `data` object.